launchable/test_runners/file.py

```diff
@@ -0,0 +1,49 @@
+#
+# The most bare-bone versions of the test runner support
+#
+import click
+import sys
+from . import launchable
+from junitparser import TestCase, TestSuite
+from ..testpath import TestPath
+
+
+@launchable.subset
+def subset(client):
+    # read lines as test file names
+    for t in sys.stdin:
+        client.test_path(t)
+    client.run()
+
+
+@click.argument('reports', required=True, nargs=-1)
+@launchable.record.tests
+def record_tests(client, reports):
+    def path_builder(case: TestCase, suite: TestSuite, report_file: str) -> TestPath:
```

Comment:
I think it's better than `CaseEvent.default_path_builder`. Do you plan to copy this `path_builder` logic to `CaseEvent.default_path_builder` or remove `CaseEvent.default_path_builder`?

Reply:
This logic builds a different looking test path compared to `default_path_builder`. I cannot replace the latter by the former without risking breaking every other test runner support. In effect, default path builder is untouchable.

I've mentioned to you in the past that the whole notion of default path builder is just fundamentally error prone because of that. So I'm +1 for removing the default path builder but for an entirely different reason.

I think we should talk. There's a disconnect here.